projects/lights_control/inc/lights_events.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+typedef enum {
+  // front board events
+  EVENT_HORN = 0,
+  EVENT_HEADLIGHTS,
+  // rear board events
+  EVENT_BRAKES,
+  EVENT_STROBE,
+  // both boards
+  EVENT_SIGNAL_LEFT,
+  EVENT_SIGNAL_RIGHT,
+  EVENT_SIGNAL_HAZARD,
+  // can internal events
+  EVENT_CAN_RX,
+  EVENT_CAN_TX,
+  EVENT_CAN_FAULT,
+  NUM_EVENTS
+} InputEvent;
```

Comment:
This should probably be called `LightControlEvents` or something, and be exported so driver controls can access. You're directly unpacking these received messages, so they need to be shared.